docs/sources/flow/reference/components/discovery.azure.md

```diff
@@ -0,0 +1,115 @@
+---
+title: discovery.azure
+---
+
+# discovery.azure
+
+`discovery.azure` discovers [Azure][] Virtual Machines and exposes them as targets.
+
+[Azure]: https://azure.microsoft.com/en-us
+
+## Usage
+
+```river
+discovery.azure "LABEL" {
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name                | Type       | Description                                                            | Default              | Required
+------------------- | ---------- | ---------------------------------------------------------------------- | -------------------- | --------
+`environment`       | `string`   | Azure environment.                                                     | `"AzurePublicCloud"` | no
+`port`              | `number`   | Port to be appended to the `__address__` label for each target.        | `80`                 | no
+`subscription_id`   | `string`   | Azure subscription ID.                                                 |                      | no
+`refresh_interval`  | `duration` | Interval at which to refresh the list of targets.                      | `1m`                 | no
```

Comment:
Should this be 1m or 5m? The code defaults to 5m.

Reply:
Good catch, updating now!